src/main/java/snowy/data/SnowyException.java

```diff
@@ -0,0 +1,7 @@
+package snowy.data;
+
+public class SnowyException extends Exception {
```

Comment:
Seems like snowy exception is reused for all different exceptions that arise from different input commands. It would be better if different types of exceptions are declared such as DeadlineDateMissingException.